lib/ack_tracker.h

```diff
@@ -26,7 +26,12 @@
 #define ACK_TRACKER_H_INCLUDED
 
 #include "syslog-ng.h"
-#include "logsource.h"
+#include "ack-type.h"
+#include "bookmark.h"
+
+typedef struct _AckTracker AckTracker;
+typedef struct _AckRecord AckRecord;
+typedef struct _LogSource LogSource;
 
```

Comment:
why isn't this in logsource.h? this file includes that anyway and seems like a better fit.

Reply:
seems you are already doing in a subsequent patch, sorry for the noise.